plutus-core/plutus-ir/test/TestLib.hs

```diff
@@ -7,6 +6,9 @@
 {-# LANGUAGE TypeApplications      #-}
 {-# LANGUAGE TypeOperators         #-}
 {-# LANGUAGE UndecidableInstances  #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Hmm, I think we do generally use the `-W`/`-w` forms for warning options, I guess we could always use `-f`... but at the moment I think we fairly consistently use the other form.

Reply:
>  I think we do generally use the -W/-w forms for warning options

That's not what `git grep` says!

Reply:
You're right, we're wildly inconsistent. Sigh.

Reply:
I've always used `-f`. Didn't even know other people used something else.

Reply:
Should we agree on a thing and make it consistent? I can replace everything here but maybe another PR is more tidy.